### PR TITLE
Improvements on the docs set-up file for new contributors

### PR DESCRIPTION
**Pull request by gvasquezvargas**
No Jira ticket, but getting ready for a new colleague joining us. I made some proposals based on my own experience, so please add/change anything as necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,18 +2,26 @@
 
 # D2iQ documentation
 
+D2IQ's documentation is created based on a *docs as code* approach or methodology. In other words, content creators use the same processes and tools as developers. Therefore, there are a few steps you need to follow to be able to contribute to documentation.
```
Comment (msdolbey) [inline README.md:5]:
```suggestion
D2iQ's documentation is created based on a *docs as code* approach or methodology. In other words, content creators use the same processes and tools as developers. Therefore, there are a few steps you need to follow to be able to contribute to documentation. 
```

Comment (msdolbey) [reply README.md:5]: just going to commit this to pass tests.

File: README.md [hunk 2]
```diff
@@ -2,18 +2,26 @@
...
-1. Clone the repo
+1. Download and install a source code editor like VSC. Note: use a package manager such as Homebrew to install new software.
+1. Create an account on GitHub and activate the 2FA process for added security. Or use your personal account and ensure 2FA is active.
+1. Install the GitHub extension (in VSC: https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github)
+1. Clone the repo (in VSC: select *Clone repo*, enter *mesosphere/dcos-docs-site* in the search bar and save a copy of the repo locally).
 1. Install node.js version 12.22.7 (consider using a version manager such as [ASDF](https://github.com/asdf-vm/asdf) or [nvm-windows](https://docs.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-windows))
 1. Windows users will need to [install python2](https://github.com/nodejs/node-gyp/tree/v3.8.0#on-windows) globally
 1. Install dependencies via command `npm i`
 1. Install [vale](https://docs.errata.ai/vale/install) (an [ASDF plugin](https://github.com/osg/asdf-vale) is also available).
+1. Your GitHub profile need to be validate with a GPG certificate.
```
Comment (msdolbey) [inline README.md:19]:
```suggestion
1. Your GitHub profile needs to be validate with a GPG certificate. 
```

Comment (msdolbey) [inline README.md:19]:
```suggestion
1. Your GitHub profile needs to be validate with a GPG certificate. 
```

**Review by msdolbey — APPROVED** [review]
👍 

loved it

File: README.md (hunk 2 above)
Comment (gvasquezvargas) [inline README.md]:
```suggestion
1. Your GitHub profile need to be validated with a GPG certificate. 
```

File: README.md [hunk 3]
```diff
@@ -2,18 +2,26 @@
...
 
+D2iQ's documentation is created based on a *docs as code* approach or methodology. In other words, content creators use the same processes and tools as developers. Therefore, there are a few steps you need to follow to be able to contribute to documentation. 
+
 D2iQ's documentation site uses a JavaScript static site generator called Metalsmith. Here is how you can get started running the repo. Please also read the [contributing guidelines](./CONTRIBUTING.md) for instructions on how to make edits to docs.
 
 ## Prerequisites
 
-1. Clone the repo
+1. Download and install a source code editor like VSC. Note: use a package manager such as Homebrew to install new software.
+1. Create an account on GitHub and activate the 2FA process for added security. Or use your personal account and ensure 2FA is active.
```
Comment (gvasquezvargas) [inline README.md:12]:
```suggestion
1. Create an account on GitHub and activate the 2FA process for added security, or use your personal account and ensure 2FA is active.
```

File: README.md [hunk 4]
```diff
@@ -2,18 +2,26 @@
...
+1. Install the GitHub extension (in VSC: https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github)
```
Comment (gvasquezvargas) [inline README.md:13]:
```suggestion
1. Add the GitHub extension to your editor (in VSC: https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github)
```

**Review by cmastr — APPROVED** [review]

**Pull request merged**